telescope/icon.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Are there icon guidelines around somewhere? I might give it a try myself.

Reply:
I didn't find any guidelines for icons. You could use the icon.svg as base. It's from the default ubuntu touch icon template. But you could check https://snwh.org/suru.

Reply:
I've found the guidelines :-) https://samuelhewitt.com/ubuntu-icon-design-guide

Reply:
Thanks. Here is my modest attempt at following the guidelines you sent:
https://www.dropbox.com/s/st3rjd4ns2k3lc4/telescope-icon-test.svg?dl=0

This icon is vaguely based on https://github.com/snwh/ubuntu-icon-resource-kit/blob/master/examples/irregular-pictogram%2Bshadows.svg which is licensed under CC BY 4.0. My icon can be used under the same terms.

Reply:
Nice thank you